File: src/main/java/com/flexpoker/util/PasswordUtils.java
package com.flexpoker.util;

import java.util.Map;

import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
import org.springframework.security.crypto.password.DelegatingPasswordEncoder;

public class PasswordUtils {

    public static final DelegatingPasswordEncoder encoder = new DelegatingPasswordEncoder(
            "bcrypt", Map.of("bcrypt", new BCryptPasswordEncoder()));

    public static String encode(String password) {
        return encoder.encode(password);
    }

}

File: src/main/kotlin/com/flexpoker/util/PasswordUtils.kt
package com.flexpoker.util

import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder
import org.springframework.security.crypto.password.DelegatingPasswordEncoder

val passwordEncoder = DelegatingPasswordEncoder("bcrypt", mapOf("bcrypt" to BCryptPasswordEncoder()))

fun encodePassword(password: String): String {
    return passwordEncoder.encode(password)
}
